JanePhotoBrowser/JanePhotoBrowser/Classes/PhotoBrowserView.swift

```diff
@@ -163,8 +184,9 @@ public class PhotoBrowserView: UIView {
         }
         
         self.imageLabel.textColor = UIColor(red: 0.2, green: 0.2, blue: 0.2, alpha: 1)
-        self.imageLabel.text = "\(row) of \(max)"
+        self.imageLabel.text = "\(index ?? row) of \(max)"
         self.imageLabel.accessibilityIdentifier = "JanePhotoBrowser-imageCountLabel"
+        self.setSmallCellSelected(at: row - 1)
```

Comment:
Should this use the `index ?? row` as well?

Reply:
This one shouldn't because we're only using the row to determine what should be highlighted. The other one needs it just to update the label